src/lib/metrics/MetricReport.ts

```diff
@@ -0,0 +1,34 @@
+import { SessionManager } from '../session/SessionManager'
+import * as pkg from '../../../package.json'
+
+export interface Metric {
+  [metricName: string]: number | string | MetricEnv
+}
+
+interface MetricEnv {
+  command: string
+  account: string
+  workspace: string
+  toolbeltVersion: string
+  nodeVersion: string
+  platform: string
+}
+
+export interface MetricReport extends Metric {
+  env: MetricEnv
+}
+
+export function metricToMetricReport(metric: Metric): MetricReport {
```

Comment:
How about using OOP here? Like on `ErrorReport`? It's optional though, the way it is is ok by me, it's just an suggestion so in the future we could standardize toolbelt to use OOP

Reply:
I did think about this, but it didn't seen necessary. I understand the argument of making it standard, though, so I will give it a try.